gui/schedule_gui.cc

```diff
@@ -810,27 +810,26 @@ bool schedule_gui_t::infowin_event(const event_t *ev)
 bool schedule_gui_t::action_triggered( gui_action_creator_t *comp, value_t p)
 {
 DBG_MESSAGE("schedule_gui_t::action_triggered()","comp=%p combo=%p",comp,&line_selector);
-
+	// for updating tool after any action happened
+	bool is_updating_tool = true;
```

Comment:
`is_updating_tool` だと違う意味になるので、 `should_set_schedule_tool` とかでしょうか。`update_tool` の引数自体がわかりづらいので名前をつけるの難しいですが...
```suggestion
	// Always call update_tool for any actions to prevent unexpected state
	bool should_set_schedule_tool = true;
```

Reply:
対応しました